tools/batch-script/cmd/get_contracts/main.go

```diff
@@ -17,31 +17,45 @@
  */
 
 // Get all contracts from a network and write them as a CSV file to standard output.
-// The CSV file has the header: address,name,code
+// The CSV file has the header: location,code
 package main
 
 import (
 	"context"
 	"encoding/csv"
 	"flag"
 	"os"
+	"time"
 
 	"github.com/onflow/cadence"
-	"github.com/onflow/cadence/tools/batch-script"
+	"github.com/onflow/cadence/runtime/common"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
+
+	"github.com/onflow/cadence/tools/batch-script"
 )
 
-var url = flag.String("u", "", "Flow Access Node URL")
+var urlFlag = flag.String("u", "", "Flow Access Node URL")
+var pauseFlag = flag.String("p", "", "pause duration")
 
-var csvHeader = []string{"address", "name", "code"}
+var csvHeader = []string{"location", "code"}
 
 func main() {
 	flag.Parse()
 
 	config := batch_script.DefaultConfig
-	if *url != "" {
-		config.FlowAccessNodeURL = *url
+
+	url := *urlFlag
+	if url != "" {
+		config.FlowAccessNodeURL = url
+	}
+
+	pause, err := time.ParseDuration(*pauseFlag)
```

Comment:
Is the flag mandatory?

Reply:
Good catch! No, it was meant to be optional, have to check it's empty

Reply:
Fixed in aaadc9eee337b9c69d6bfb254ca2ab91683ef388